man/fi_mr.3.md

```diff
@@ -736,6 +743,11 @@ The follow flag may be specified to any memory registration call.
   API AllocHost call (as opposed to using malloc-like host allocation,
   unified/shared memory allocation, or AllocDevice).
 
+*FI_HMEM_DATA_GDRCOPY_HANDLE*
+: For libfabric internal use only. This flag indicates that hmem_data points to
+  a gdrcopy_handle data structure. This flag is ignored unless iface is set to
+  FI_HMEM_CUDA.
+
```

Comment:
Thanks. Addressed comments. Stripped away unrelated changes as well since we are not exposing the new `OFI_HMEM_*` flag. This makes the change minimal.

Reply:
The CI appears to be down recently. Assuming the failure is unrelated.